src/Microsoft.ApplicationInsights.AspNetCore/TelemetryInitializers/AzureWebAppRoleEnvironmentTelemetryInitializer.cs

```diff
@@ -11,7 +10,7 @@
     /// <summary>
     /// A telemetry initializer that will gather Azure Web App Role Environment context information.
     /// </summary>
-    public class AzureWebAppRoleEnvironmentTelemetryInitializer : TelemetryInitializerBase
+    internal class AzureWebAppRoleEnvironmentTelemetryInitializer : TelemetryInitializerBase
```

Comment:
My idea was to only expose API's that we are sure about in this release, so if we decide to rearrange things internally that would not count as another breaking change.